airflow/cli/cli_parser.py

```diff
@@ -2087,7 +2101,10 @@ def get_long_option(arg: Arg):
         """Get long option from Arg.flags"""
         return arg.flags[0] if len(arg.flags) == 1 else arg.flags[1]
 
-    positional, optional = partition(lambda x: x.flags[0].startswith("-"), args)
+    def __flags(x):
+        return x.flags[0].startswith("-")
+
+    positional, optional = partition(__flags, args)
```

Comment:
Why this change? Seems equivalent to me

Reply:
emmm，It was modified during debugging and has been adjusted back...